node/src/types/committee_mode.rs

```diff
@@ -0,0 +1,51 @@
+use bytes::BytesMut;
+use postgres_types::Json;
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use std::error::Error;
+use tokio_postgres::types::{accepts, to_sql_checked, FromSql, IsNull, ToSql, Type};
+
+#[derive(Clone, Serialize, PartialEq, Debug, Deserialize)]
+pub enum NodeSelectionStrategy {
+    RegisterAll = 1,
```

Comment:
Just put this as a temporary value here, wasn't sure what strategies we would support.

Reply:
Yeah, I stubbed other types too, we can amend implementation later, the idea though is that we should use type methods, rather than accessing value directly so that we free to change type in the future.